src/Twig/MediaRuntime.php

```diff
@@ -140,11 +140,11 @@ public function path($media, string $format): string
     }
 
     /**
-     * @param MediaInterface|int|string $media
+     * @param int|string|MediaInterface|null $media
      */
     private function getMedia($media): ?MediaInterface
     {
-        if (!$media instanceof MediaInterface) {
+        if (!$media instanceof MediaInterface && null !== $media) {
```

Comment:
we might deprecate after this PR the posibility to pass null here. wdyt @sonata-project/contributors ?

IMO we should allow it on 3.x (to not complicate more the update) but with a deprecation message, and remove it on 4.x)

Reply:
Agree, i dont find natural to support null

Reply:
Yep, it is needed to introduce it on 3.x again, because otherwise we are breaking BC, but we can remove it on 4.x for sure. I will prepare another PR to deprecate it.